content/fr/c19-research.html

```diff
@@ -2,27 +2,31 @@
+            <h2 class="section--title">Est-ce que vous ou une personne de votre entourage vivez un stress financier en raison de la COVID-19?</h2>
+            <p>Notre équipe au gouvernement du Canada est actuellement en train d’améliorer notre site Web sur la COVID-19. Nous mènerons des séances de recherche au cours des prochaines semaines afin de mesurer à quel point notre site Web est utile pour les personnes vivant un stress financier.</p>
+            <p>Les séances de recherche auront lieu par vidéoconférence et dureront environ 30 minutes. Il n’est pas nécessaire d’avoir une expertise liée à la COVID-19.</p>
+        <hr>
+            <h2>Comment participer</h2>
+            <p>Si vous souhaitez vous porter volontaire pour notre étude,</p>
+        <ol>
+            <li>Contactez Philippe Caron à l’adresse <a href="mailto:philippe.caron@tbs-sct.gc.ca">philippe.caron@tbs-sct.gc.ca</a> pour exprimer votre intérêt.</li>
+            <li>Nous vous poserons ensuite quelques questions préliminaires pour déterminer comment vous pourriez contribuer à notre recherche.</li>
+            <li>Nous pourrions alors vous contacter pour que vous participiez à une séance.</li>
+            </ol>
+            
             <p>En participant à cette recherche, vous comprenez que :</p>
             <ul>
                 <li>Votre participation est volontaire. Vous pouvez arrêter de participer en tout temps et peu importe la raison, sans aucune conséquence.</li>
                 <li>Vos renseignements seront anonymes, ce qui signifie que vos réponses ne seront pas reliées à vous.</li>
                 <li>Pour cette raison, vous ne pourrez pas retirer ou corriger vos rensegnements une fois que vous les aurez fournis.</li>
                 <li>Dans vos réponses, veuillez ne pas fournir de renseignements permettant de vous identifier.</li>
                 <li>Votre participation et vos réponses n’affecteront pas votre accès aux services ou aux prestations du gouvernement du Canada.</li>
-                <li>Nous utiliserons Zoom pour la session. Voici un lien vers leur politique concernant les données confidentielles : <a href="https://zoom.us/fr-fr/privacy.html">https://zoom.us/fr-fr/privacy.html</a></li>
-                <li>Le SNC et EDSC peuvent recueillir des renseignements personnels comme vos antécédents professionnels et votre expérience des services publics. Pour vous renseigner sur la façon dont le SNC et EDSC protègent vos renseignements personnels, veuillez consulter l’avis de confidentialité ci-joint.</li>
+                <li>Le Service numérique canadien (SNC)peut recueillir des renseignements personnels comme vos antécédents professionnels et votre expérience des services publics. Pour vous renseigner sur la façon dont le SNC protège vos renseignements personnels, veuillez consulter l’avis de confidentialité ci-joint.</li>
```

Comment:
just missing a space after (SNC)
```suggestion
                <li>Le Service numérique canadien (SNC) peut recueillir des renseignements personnels comme vos antécédents professionnels et votre expérience des services publics. Pour vous renseigner sur la façon dont le SNC protège vos renseignements personnels, veuillez consulter l’avis de confidentialité ci-joint.</li>
```

Reply:
All looks good to me!